hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -201,8 +202,11 @@ public final class Hedera implements SwirldMain {
      * improve testability and reuse. (For example, the {@link BootstrapConfigProviderImpl}.)
      *
      * @param constructableRegistry the registry to register {@link RuntimeConstructable} factories with
+     * @param registry the registry to register services with. This is optional and can be null.
+     *                         If null, a new instance of {@link ServicesRegistryImpl} will be created.
+     *                         This is useful for testing.
      */
-    public Hedera(@NonNull final ConstructableRegistry constructableRegistry) {
+    public Hedera(@NonNull final ConstructableRegistry constructableRegistry, @Nullable ServicesRegistryImpl registry) {
```

Comment:
I suggest using the interface type here, i.e. `@NonNull ServicesRegistry`; and we will pass a `FakeServicesRegistry` to be implemented later.

Reply:
Fixed @tinker-michaelj